contracts/voting-verifier/src/execute.rs

```diff
@@ -63,16 +62,16 @@ pub fn verify_verifier_set(
 
     POLL_VERIFIER_SETS.save(
         deps.storage,
-        &new_operators.hash(),
-        &PollContent::<Operators>::new(new_operators.clone(), poll_id),
+        &new_verifier_set.hash().as_slice().try_into().unwrap(),
```

Comment:
we wanted to change the hash implementation since it's just serializing to JSON rn and hashing that. This would be a breaking change later